### ISSUE TITLE
Logs are missing query strings

### ISSUE THREAD

**Issue by marfire** [Miserlou/Zappa#1048]
## Context
The logs produced by Zappa are crippled by the fact that they don't include the query string.

## Expected Behavior
`"GET /song?title=ship+ahoy HTTP/1.1"`

## Actual Behavior
`"GET /song HTTP/1.1"`

## Possible Fix
This appears to be [a bug](https://github.com/pklaus/wsgi-request-logger/issues/13) in `wsgi-request-logger`.

It could be easily fixed, but that project seems rather abandoned (10 months of no activity). [This bug](https://github.com/pklaus/wsgi-request-logger/issues/12) is also of note. :wink:

I'm not sure what step is best for the Zappa project, but this functionality probably needs to be replaced by a different project or forked by / absorbed into Zappa.

**Comment by Miserlou** [comment]
Ahhhhhh. Good catch. Hm. Try poking upstream with a PR?

**Comment by marfire** [comment]
I left a PR a few months ago, but there hasn't been any activity on that project in 15 month. Do you have an idea of how you'd like to handle dependencies that are abandoned?

**Comment by Miserlou** [comment]
Hm. We could absorb the code and maintain it internally, that particular one has been problematic in the past as well. That's not very preferable though.

**Comment by marfire** [comment]
I agree, it would be better to keep it as a separate project if possible.

I wonder if, as Zappa grows, there are parts of its codebase that can be spun out into separate projects? I think there are a lot of advantages to that approach: more people can reuse the code, you get more eyeballs on it, more contributions, and faster release cycles. If you wanted to go that route, you could create a Zappa organization and put projects under it (like this one).

If not, I'd be willing to transfer or fork it to my own account and maintain it.
